alacritty/src/config/monitor.rs

```diff
@@ -10,6 +10,11 @@ use alacritty_terminal::thread;
 
 use crate::event::{Event, EventProxy};
 
+#[cfg(any(target_os = "linux", target_os = "macos", target_os = "windows"))]
+const DEBOUNCE_DELAY: Duration = Duration::from_millis(10);
```

Comment:
Does this delay rely matter for linux/macos/windows? Could just have 1 sec for everything.

Reply:
Yes, the debounce period applies to linux/macos/windows too. With the file watcher it's the frequency for which the file is polled, with inotify it's the time for which an event is held before it is dispatched, to make sure operations that happen together don't unnecessarily trigger things twice.